.github/ISSUE_TEMPLATE/J-Enviar-consulta-pré-submissão

```diff
@@ -0,0 +1,69 @@
+---
+name: Enviar uma consulta de pré-submissão
+about: Não tem certeza de que seu software se encaixa ou está enviando um pacote estatístico? Use este modelo para obter uma resposta rápida dos editores
+
+---
+
+<!---
+Below, please enter values for (1) submitting author GitHub handle (replacing "@github_handle"); and (2) Repository URL (replacing "https://repourl"). Values for additional package authors may also be specified, replacing "@github_handle1", "@github_handle2" - delete these if not needed. DO NOT DELETE HTML SYMBOLS (everything between "<!" and ">"). Replace only "@github_handle" and "https://repourl". This comment may be deleted once it has been read and understood.
+--->
+
+Submitting Author Name: Name
+Submitting Author Github Handle: <!--author1-->@github_handle<!--end-author1-->
+Other Package Authors Github handles: (comma separated, delete if none) <!--author-others-->@github_handle1, @github_handle2<!--end-author-others-->
+Repository:  <!--repourl-->https://repourl<!--end-repourl-->
+Submission type: <!--submission-type-->Pre-submission<!--end-submission-type-->
+Language: <!--language-->en<!--end-language-->
+
+---
+
+-   Paste the full DESCRIPTION file inside a code block below:
+
+```
+
+```
+
+
+## Scope 
+
+- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) or [statistical package categories](https://stats-devguide.ropensci.org/overview.html#overview-categories) this package falls under. (Please check one or more appropriate boxes below):
```

Comment:
```suggestion
- Por favor, indique a(s) categoria(s) que se aplicam ao pacote. É possível consultar as categorias na nossa [política de revisão por pares](https://ropensci.github.io/dev_guide/policies.html#package-categories) ou nas [categorias de pacotes estatísticos](https://stats-devguide.ropensci.org/overview.html#overview-categories). Marque as opções apropriadas abaixo:
```

In this case, I used the Spanish version as reference, so the text is more fluid.
https://raw.githubusercontent.com/ropensci/software-review/refs/heads/main/.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md